### DISCUSSION TITLE
JSON querying issues

### DISCUSSION THREAD

**Discussion by lpar** [mikefarah/yq#1351] [Q&A] [answered]
How can I write four commands to extract all four values individually from the following simple JSON example?

```json
{
  "key": "alpha",
  "another.key": "beta",
  "another": {
    "key": "gamma"
  },
  "key.three": "delta"
}
```

**Comment by mikefarah** [comment, marked as answer]
Because some of the keys have dots (.) in them, you will need to use the special character syntax documented here: https://mikefarah.gitbook.io/yq/operators/traverse-read#special-characters

**Comment by lpar** [comment]
Awesome, thanks. It hadn't occurred to me to look under "Special characters" because I didn't think of `.` as a special character.
